src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -62,7 +70,7 @@ class BaseCreateCourseForm extends React.Component {
           <Field
             name="org"
             component={RenderSelectField}
-            options={this.processOrganizations(organizations)}
+            options={this.state.organizations}
```

Comment:
(Similar to above) Not sure if this change is necessary in order to default to the only organization in the list